src/golang/lib/repos/user.go

```diff
@@ -16,21 +16,17 @@ type User interface {
 
 type userReader interface {
 	// GetByAPIKey returns the User with the API key apiKey.
+	// It returns a database.ErrNoRows if no rows are found.
 	GetByAPIKey(ctx context.Context, apiKey string, DB database.Database) (*models.User, error)
-
```

Comment:
Why did we remove this?

Reply:
`GetOrgAdmin` was unused and for now the open source doesn't have any admin users, it just has a single user.